### ISSUE TITLE
turn the effect list documentation into markdown

### ISSUE THREAD

**Issue by Helianthella** [HerculesWS/Hercules#2215]
`doc/effect_list.txt` should be rewritten in markdown for easier reading in the browser.

proposed syntax:

```markdown
# Effect list

(description)


effect | description
-------|------------
0      | regular hit
1      | bash
2      | melee skill hit
3      | melee skill hit

...
```

**Comment by yurrr** [comment]
Hi, I'll be working on this :D

**Comment by dastgirp** [comment]
merged #2230 , closing this
